--- criterion-2.4.2/meson.build.orig	2023-09-01 10:21:07.575847344 +0200
+++ criterion-2.4.2/meson.build	2023-09-01 16:43:34.211113324 +0200
@@ -181,7 +181,7 @@ else
 endif
 
 nanopb = dependency('nanopb', required: get_option('wrap_mode') == 'nofallback', method: 'cmake',
-					modules: ['nanopb::protobuf-nanopb-static'])
+					modules: ['nanopb::protobuf-nanopb'])
 must_regenerate_pb = nanopb.found()
 if (not nanopb.found()
 	or get_option('wrap_mode') == 'forcefallback'
