--- criterion-2.4.1/subprojects/boxfort/meson.build.orig	2022-04-26 08:55:02.322888100 +0200
+++ criterion-2.4.1/subprojects/boxfort/meson.build	2022-11-20 00:30:33.386292493 +0100
@@ -81,7 +81,13 @@
 if arch == 'x86'
 	bitness = 32
 elif arch == 'x86_64'
-	bitness = 64
+	if cc.compiles('''#ifdef __ILP32__
+#error x32
+#endif''')
+		bitness = 64
+        else
+		bitness = 32
+        endif
 elif arch == 'arm'
 	bitness = 32
 elif arch == 'aarch64'
